src/components/ItemCard.js

```diff
@@ -0,0 +1,199 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import CardSlide from './CardSlide';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+
+const ItemCard = () => {
+  const [roomData, setRoomData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/RoomData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => setRoomData(data));
+  });
+
+  return (
+    <Card>
+      {roomData.map(data => {
+        return (
+          <>
+            <CardImage>
+              <CardSlide />
+            </CardImage>
+            <CardText key={data.host_id}>
+              <Link to="/login">
+                <CardTitle>
+                  <TxtTitle>{data.name}</TxtTitle>
+                  <CardGrade>
+                    <i className="fa-solid fa-star" />
+                    &nbsp;5.0&nbsp;(6)
+                  </CardGrade>
+                </CardTitle>
+                <Descrition>{data.address}</Descrition>
+                <Descrition>{data.description}</Descrition>
+                <Descrition>침대 {data.bed}개</Descrition>
+                <CardPrice>
+                  <DayPrice>￦{data.price.toLocaleString()} /박</DayPrice>
+                  <TotalPrice>총액 ￦120,150</TotalPrice>
+                </CardPrice>
+              </Link>
+            </CardText>
+          </>
+        );
+      })}
+    </Card>
+  );
+};
+
+const Card = styled.div`
+  display: inline-block;
+  margin-right: 24px;
+  margin-bottom: 40px;
+  width: 16.875vw;
+  min-width: 320px;
+  a {
+    text-decoration: none;
+    line-height: 1.4;
+  }
```

Comment:
```suggestion
  const xxx = styled(Link)`
  text-decoration: none;
  line-height: 1.4;
`;
```
Link 컴포넌트에 대한 스타일링을 주시려 사용하신 것 같은데
스타일 컴포넌트에서는 이런 방식으로도 스타일 적용이 가능합니다
변수명은 직접 생각해 보세요!